public/locales/en-US/translations.json

```diff
@@ -364,8 +364,8 @@
   "only_xrp":"Only XRP",
   "trust_line":"Creates Trust Lines for Issuer",
   "transferable":"Transferable",
-  "buy_offers":"Buy Offers",
-  "sell_offers":"Sell Offers",
+  "buy-offers":"Buy Offers",
+  "sell-offers":"Sell Offers",
```

Comment:
why did you change to a dash?

Reply:
Because we are using the same string for locale and the url inside Tabs component. And I dont think we can have underscore in url by convention

`const Tabs = (props: Props) => {
  const { t } = useTranslation()
  const { tabs, selected, path } = props
  const items = tabs.map((title) => {
    const className = selected === title ? 'tab selected' : 'tab'
    return (
      <Link
        className={className}
        title={t(title)}
        key={title}
        to={`${path}/${title}`}
      >
        {t(title)}
      </Link>
    )
  })
  return <div className="tabs">{items}</div>
}`

Reply:
you could do `t(title.replaceAll('-', '_'))`

Reply:
i dont think replaceAll is supported in some browser versions lol. And also my tests is throwing error about not recognzing replaceAll 🤣. ill find an alternative